smithy-model/src/main/java/software/amazon/smithy/model/knowledge/ServiceIndex.java

```diff
@@ -105,6 +105,8 @@ private Model getModel() {
      * <p>An <em>auth defining trait</em> is a trait that is marked with
      * the {@code smithy.api#authDefinition} trait.
      *
+     * <p>The returned map is ordered alphabetically by absolute shape id.
```

Comment:
nit: capitalize "id"

```suggestion
     * <p>The returned map is ordered alphabetically by absolute shape ID.
```

Reply:
Fixed. Also fixed some other ones from the previous PR